packages/verkle/src/verkleTree.ts

```diff
@@ -250,11 +250,28 @@ export class VerkleTree {
         )
       }
     } else {
+      if (equalsBytes(value, createDeletedLeafValue())) {
```

Comment:
Should this not throw an error instead of silently returning?

Reply:
Now that I've re-read the spec, I'm not sure what the expected behavior is.  If it's just writing `0` to the trie, then this behavior is actually invalid and we should just let it create a new leaf node and write what I'm calling the `deleted leaf value` here.  Will ask kev

Reply:
My instinct here is that we should actually just create a new leaf node and write zeroes to it.  There is no "deletion" in verkle tries so writing zeroes to a leaf value is the same as "deleting" a value in the trie.  Will update this section of code accordingly (i.e. just remove this special handling and allow `trie.del to write zeroes)

Reply:
Ok